pkg/controller/controller_instance.go

```diff
@@ -1408,11 +1408,12 @@ func (c *controller) prepareRequestHelper(instance *v1beta1.ServiceInstance, ser
 
 	// osb client handles whether or not to really send this based
 	// on the version of the client.
+	id := c.getClusterID()
 	rh.requestContext = map[string]interface{}{
 		"platform":  ContextProfilePlatformKubernetes,
 		"namespace": instance.Namespace,
+		"clusterID": id,
```

Comment:
if this is the json payload for request context, I think `clusterID` should be `cluster_id` to be more standard OSB style. I think this was an oversight on the OSB side. https://github.com/openservicebrokerapi/servicebroker/issues/477

Reply:
yep - its now `clusterid`, https://github.com/openservicebrokerapi/servicebroker/pull/479

Reply:
changed.